netdeploy/network.go

```diff
@@ -146,6 +147,7 @@ func (n Network) NodeDataDirs() []string {
 	for _, nodeDir := range n.nodeDirs {
 		directories = append(directories, n.getNodeFullPath(nodeDir))
 	}
+	sort.Strings(directories)
```

Comment:
this change ensures that the output of the method is consistent; it's not so by default since the `nodeDirs` is a map.